pandas/tests/test_resample.py

```diff
@@ -2812,18 +2817,96 @@ def test_evenly_divisible_with_no_extra_bins(self):
         result = df.resample('7D').sum()
         assert_frame_equal(result, expected)
 
-    def test_apply_to_empty_series(self):
-        # GH 14313
-        series = self.create_series()[:0]
+    @pytest.mark.parametrize('kind', ['period', None, 'timestamp'])
+    @pytest.mark.parametrize('agg_arg', ['mean', {'value': 'mean'}, ['mean']])
+    def test_loffset_returns_datetimeindex(self, frame, kind, agg_arg):
+        # make sure passing loffset returns DatetimeIndex in all cases
+        # basic method taken from Base.test_resample_loffset_arg_type()
```

Comment:
This is OK-but-not-ideal behavior - I think pandas' approach is to write tests around that behavior but also to note that. 
Something like "TODO: not ideal to convert to DatetimeIndex, but...", and maybe an issue number

Reply:
this is ok for now, IIRC this is on the master PR list already?